stylus/include/AboutUs.styl

```diff
@@ -0,0 +1,44 @@
+.AboutUs
+	>div:nth-child(1)
+		display: flex
+		padding-bottom: 6rem
+		padding-top: 4rem
+		>div
+			width: 50%
+			>img
+				padding-left: 3rem
+				padding-right: 3rem
+			>div
+				margin-right: 5rem
+			>div:nth-child(1)
+				font-family: Arvo
+				font-size: 2rem
+			>div:nth-child(8)
+				font-weight: bold
+				text-transform: uppercase
+		>div:nth-child(2)
+			padding-top: 3rem
+	>div:nth-child(2)
+		display: flex
+		flex-wrap: wrap
```

Comment:
Looking at the site, you probably don't want this to wrap. It seems that clicking an arrow will reveal other images, so it should just be one row here.